examples/clients/clients.py

```diff
@@ -9,7 +9,8 @@
 # Retrieve email statistics by client type. #
 # GET /clients/stats #
 
-params = {'aggregated_by': 'day', 'start_date': '2016-01-01', 'end_date': '2016-04-01'}
+params = {'aggregated_by': 'day',
```

Comment:
Would it be ok to be pedantic here and ask that end_date be moved to the next line too?

Reply:
Updated.